packages/story-editor/src/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -127,6 +131,14 @@ function PageTemplatesPane(props) {
     setNextTemplatesToFetch,
   ]);
 
+  const handleToggle = (evt, value) => {
+    setShowDefaultTemplates(value === DEFAULT);
+    window.localStorage.setItem(
```

Comment:
Import `localStore` from `@web-stories-wp/design-system` instead and use `localStore.getItemByKey` and `localStore.getItemByKey`/`localStore.setItemByKey` to get error handling for free. 

There's also `LOCAL_STORAGE_PREFIX`

Reply:
Nice, Thanks @swissspidy